### PR TITLE
Create 2023-Q2-Hyperledger-Besu.md

### PR DESCRIPTION
**Pull request by non-fungible-nelson**

### CODE REVIEW

**Review by petermetz — APPROVED** [review]

File: project-reports/2023/2023-Q2-Hyperledger-Besu.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+# Questions/Issues for the TOC
+
+Have we been able to complete the asks around CircleCI funding by Hyperledger Foundation?
+
+# Releases
+
+There were only three Besu releases this quarter, two point releases with fixes and a huge quarterly with a number of new changes. We expect another rather large quarterly in July with a huge amount of features focused on performance, fixes, and tech debt. We are also using these quarterlies to deprecate older features relating to Quorum, which is no longer being maintained by Consensys.
```
Comment (jimthematrix) [inline project-reports/2023/2023-Q2-Hyperledger-Besu.md]: great to hear about the significant releases. might be a good idea if you can add a bit details on "older features relating to Quorum". Thanks!

**Review by jimthematrix — APPROVED** [review]

File: project-reports/2023/2023-Q2-Hyperledger-Besu.md (hunk 1 above)
Comment (non-fungible-nelson) [reply project-reports/2023/2023-Q2-Hyperledger-Besu.md]: We removed compatibility with GoQuorum privacy in 23.4.0 and are removing compatibility with GoQuorum-compatible permissioning in 23.7.0.

File: project-reports/2023/2023-Q2-Hyperledger-Besu.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Maintainers are quite active in engaging user questions and they often get what they need or we walk them through debugging with us so maintainers can make client improvements. We are also seeing an uptick in users helping users, which bolsters the community. The TOC may want to consider a way to track or reward these Discord "power users."
+
+The project is working towards updating our modular code base in a way that actually uses modularity as opposed to a monolith with expanding scope. There is desire to set up a working group around abstracting consensus mechanisms in Besu which should greatly help private networks take new versions of Besu into production. This should also alleviate unintended side-effects of bugs or upstream changes on the rapidly evolving Mainnet from impacting long-standing networks. Multiple contributor groups are rallying around this goal. There is also activity around Besu as a part of the rollup ecosystem which is gradually evolving. Consensys has shared some roadmap items on 2nd Half but is actively soliciting feedback from other groups on project direction.
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q2-Hyperledger-Besu.md:35]: > roadmap items on 2nd Half

Do you have a link to the roadmap that you could share?

**Review by tkuhrt — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

File: project-reports/2023/2023-Q2-Hyperledger-Besu.md (hunk 2 above)
Comment (non-fungible-nelson) [reply project-reports/2023/2023-Q2-Hyperledger-Besu.md:35]: https://wiki.hyperledger.org/display/BESU/Roadmap
updated this page

**Review by arsulegai — APPROVED** [review]

**Review by lehors — APPROVED** [review]

**Review by swcurran — APPROVED** [review]

**Review by bobbi28 — APPROVED** [review]

**Review by VRamakrishna — APPROVED** [review]

**Pull request merged**